### ISSUE TITLE
lifetime prop does not exist in animation in branch 0.8.x

### ISSUE THREAD

**Issue by BobbyBakes** [luanpotter/flame-example#4]
```Compiler message:
lib/main.dart:74:27: Error: The getter 'lifeTime' isn't defined for the class '#lib1::Animation'.
Try correcting the name to the name of an existing getter, or defining a getter or field named 'lifeTime'.
    return this.animation.lifeTime >= TIME;
                          ^^^^^^^^
Compiler failed on /Users/bob/projects/flame-example/lib/main.dart
Error launching application on iPhone XR.```

**Comment by BobbyBakes** [comment]
Maybe this just needs a proper getter?

**Comment by BobbyBakes** [comment]
Looks like this is due to upgrading flame.

**Comment by luanpotter** [comment]
Hi, @BobbyBakes , thanks for the feedback. Indeed it's outdated.

Now the animation class already has a method `.done()` that returns if it's over or not. See [here](https://github.com/luanpotter/flame/blob/master/lib/animation.dart) line 141. If'd like, I'd be glad to merge a PR with this fix to flame-example, to make it work with latest version of flame.

**Comment by luanpotter** [comment]
This was fixed by #5 ! Thanks a lot, @mcoolidge !
